File: JAVA Fundamentals/5. Basic Syntax, Conditional Statements and Loops/P01_StudentInformation.java
package BasicsSyntaxLab;

import java.util.Scanner;

public class P01_StudentInformation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String studentName = scanner.nextLine();
        int age = Integer.parseInt(scanner.nextLine());
        double grade = Double.parseDouble(scanner.nextLine());


        //"Name: {student name}, Age: {student age}, Grade: {student grade}".
        //%.2f

        System.out.printf("Name: %s, Age: %d, Grade: %.2f", studentName, age, grade);
    }
}

File: C# Fundamentals/5. Basic Syntax, Conditional Statements and Loops/P01.StudentInformation.cs
/*
John
15
5.40
*/

string studentName = Console.ReadLine();
int studentAge = int.Parse(Console.ReadLine());
double averageGrade = double.Parse(Console.ReadLine()); 

Console.WriteLine($"Name: {studentName}, Age: {studentAge}, Grade: {averageGrade:f2}");
